src/Symfony/Component/Console/Output/ConsoleOutputInterface.php

```diff
@@ -32,4 +32,11 @@ public function getErrorOutput();
      * @param OutputInterface $error
      */
     public function setErrorOutput(OutputInterface $error);
+
+    /*
+     * Creates a new output section
+     *
+     * @return ConsoleSectionOutput
+     */
+    // public function section(); // TODO Add this method to the interface in Symfony 4
```

Comment:
Let's use an `@method` annotation instead, if we really want it to be part of the interface.